modules/appeals_api/spec/support/rswag_config.rb

```diff
@@ -258,8 +258,7 @@ def schemas(api_name: nil)
         decision_reviews_sc_alt_signer_schemas,
         contestable_issues_schema,
         legacy_appeals_schema,
-        generic_schemas,
-        shared_schemas.slice(*%w[address phone timezone nonBlankString])
```

Comment:
I noticed when updating the shared schema for `address.json` (which is only used in v0 APIs) that these shared schemas were mistakenly being included in the decision reviews docs. These were duplicating and overwriting the same-named values defined in the individual decision reviews headers schemas.

As soon as I removed them from this list and regenerated the docs, the definitions of these field types (`address`, `phone`, `timezone`, `nonBlankString`) in decision reviews were updated to use their original definitions from the decision reviews schemas instead of the shared schemas. This mostly had no effect at all on the swagger docs, since the definitions were duplicates, but there was one issue: the decision reviews versions of these schemas included a `$comment` describing the `nonBlankString` field. I removed this `$comment` from the original schemas where I saw it because:
1. `$comment` is not a valid key in openapi json configs, and
2. We actually remove these `$comment`s before rendering the schemas themselves at schema endpoints in our APIs, like [here, for example](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/appeals_api/app/controllers/appeals_api/v2/decision_reviews/higher_level_reviews_controller.rb#L52)